client/reader/list-manage/list-form.jsx

```diff
@@ -63,11 +65,15 @@ export default function ListForm( { isCreateForm, isSubmissionDisabled, list, on
 					<FormLabel htmlFor="list-slug">{ translate( 'Slug (Required)' ) }</FormLabel>
 					<FormTextInput
```

Comment:
Shall we just drop this input altogether? Let's automatically create the slug for the user 🐌

Reply:
The slug input is only shown for the update form and we already create the slug automatically during list creation.

I was thinking it'd be nice to show the list slug via a disabled input for informational purposes.

Reply:
👍 As the user can't change it, shall we just show it in text rather than a disabled form input?